openreview/venue/process/comment_process.py

```diff
@@ -39,7 +41,8 @@ def process(client, edit, invitation):
 To view the comment, click here: https://openreview.net/forum?id={submission.id}&noteId={comment.id}'''
 
     program_chairs_id = domain.get_content_value('program_chairs_id')
-    if parent_invitation.content.get('email_pcs', {}).get('value', False) and (program_chairs_id in comment.readers or 'everyone' in comment.readers):
+    comment_email_pcs = parent_invitation.content.get('email_pcs', {}).get('value', domain.get_content_value('comment_email_pcs'))
```

Comment:
It looks like this wouldn't work, because we only return the default value if the invitation/group has no content; otherwise, we return the value of the field, which could be None. Looks like `get_content_value()` does not work exactly like `get()` does. I can fix this

Reply:
ohhhhh, you are right.

Reply:
Fixed!